examples/ts_js_interop/BUILD.bazel

```diff
@@ -0,0 +1,45 @@
+load("@aspect_rules_ts//ts:defs.bzl", "ts_project")
+load("@aspect_rules_swc//swc:defs.bzl", "swc")
+load("@bazel_skylib//lib:partial.bzl", "partial")
+
+ts_project(
+    name = "a_b",
+    srcs = [
+        "a.js",
+        "b.ts",
+    ],
+    allow_js = True,
+    declaration = True,
+    emit_declaration_only = True,
```

Comment:
When using `emitDeclarationOnly` the transpiler won't run at all, tsc or a custom transpiler like swc, so `b.js` should not be expected at all

Reply:
I figured as much- it seems like emit_declaration_only is not the fix to my issue and this test case can be deleted. I only included it to point out that this is one way to avoid the conflict error

Reply:
Right, I figured as much. So this test case can be deleted and clearly does not solve my problem. I only included it to point out that this is *one* way to avoid the conflict error, but I need b.js to be produced in my environment.